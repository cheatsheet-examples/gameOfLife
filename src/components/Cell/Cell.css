.Cell {
    background: #fff;
    position: absolute;
}
