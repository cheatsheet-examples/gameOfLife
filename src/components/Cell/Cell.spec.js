import React from 'react';
import { shallow } from 'enzyme';
import Cell from './Cell';

const MOCK_CELL_SIZE = 20;
const MOCK_CELL_X = 9;
const MOCK_CELL_Y = 15;

describe('Cell', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Cell
        x={MOCK_CELL_X}
        y={MOCK_CELL_Y}
        cellSize={MOCK_CELL_SIZE}
      />
    );
  });

  it('should render correctly', () => expect(wrapper).toMatchSnapshot());

  it('should render a <div />', () => {
    expect(wrapper.find('div').length).toEqual(1);
  });
});
