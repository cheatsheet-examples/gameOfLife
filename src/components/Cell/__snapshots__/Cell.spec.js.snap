// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell should render correctly 1`] = `
<div
  className="Cell"
  style={
    Object {
      "height": "19px",
      "left": "181px",
      "top": "301px",
      "width": "19px",
    }
  }
/>
`;
